Zend/zend_observer.c

```diff
@@ -229,6 +229,7 @@ ZEND_API void ZEND_FASTCALL zend_observer_fcall_end(
 		zend_execute_data *ex = execute_data->prev_execute_data;
 		while (ex && (!ex->func || ex->func->type == ZEND_INTERNAL_FUNCTION
 		          || !ZEND_OBSERVABLE_FN(ex->func->common.fn_flags)
+		          || !&RUN_TIME_CACHE(&ex->func->op_array)
```

Comment:
We thought alternatively to make sure the dummy execute_data created in `call_attribute_constructor` (`ext/reflection/php_refleciton.c`) should do what `zend_get_call_trampoline_func` does, but we didn't manage to replicate this:

```c
ZEND_MAP_PTR_INIT(func->run_time_cache, (void***)&dummy);
```